firestore/scripts/FriendlyEats.js

```diff
@@ -29,11 +40,14 @@ function FriendlyEats() { // eslint-disable-line no-redeclare
   this.dialogs = {};
 
   var that = this;
-
   firebase.firestore().enablePersistence()
     .then(function() {
```

Comment:
Not your responsibility, but I think if these were converted to arrow functions you would not need to do this `var that = this;` trick.

Reply:
Noted. I also noticed that we seem to like using `var`s on all of our scripts except for `FriendlyEats.Data`, where we're using `const`s . Is that also an inconsistency that could be cleaned up one day?